docs/how_to/tensorflow_install/tensorflow_install.md

```diff
@@ -9,6 +9,19 @@ training and inference in neural networks. It is one of the most popular and
 in-demand frameworks and is very active in open source contribution and
 development.
 
+:::{warning}
+ROCm 5.6 and 5.7 deviate from the standard practice of supporting the last three
+TensorFlow versions. This is due to incompatibilities between earlier TensorFlow
+versions and changes introduced in the ROCm 5.6 compiler. Please refer to the
```

Comment:
Please remove the word 'Please' :) Has Aswin's team reviewed this note? Please confirm.

@jithunnair-amd @amathews-amd

Reply:
cc: @jayfurmanek

Reply:
Removed :)